qemu/tests/cfg/balloon_check.cfg

```diff
@@ -60,6 +60,7 @@
         - iommu_enabled:
             only q35
             only balloon_evict_and_enlarge
+            only Windows
```

Comment:
It is better to add a comment about the product bz 1791593

Reply:
Thanks, added.